3drepo/3drepo.io
# ISSUE 2774 - Scripts for getting face count

This fixes #2774

#### Description
Python scripts for getting face count of meshes from DB

#### Test cases
- Run script with valid args should produce output
- Run script with invalid args should exit
